style.css

```diff
@@ -0,0 +1,194 @@
+  background-size: cover;
+  background-position: center;
+  height: 300px; 
+  color: #fff;
+  text-align: center;
+  margin-bottom: 10px;
+  transition: background-image 1s ease; /* Define transition for background image */
+}
+
+.sortButtons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  height: 40px;
+  padding: 5px 30px;
+}
+
+button, select, 
+#btnRate, #btnRandom{
+  font-size: 0.875em;
+  margin: 10px;
+  padding: 10px 25px;
+  background-color:rgb(255, 255, 255); 
+  color: var(--f2);
+  border: none;
+  border-radius: 20px;
+  cursor: pointer;
+  height: 100%;
+}
+
+button:hover, select:hover, 
+#btnRandom:hover, #btnRate:hover{
+  color: #fff;
+  background-color: var(--f2);
+}
+
+#az {
+  border-radius: 20px 0 0 20px ;
+  padding: 10px;
+  margin-right: 0;
+}
+
+#za{
+ border-radius: 0 20px 20px 0;
+ padding: 10px;
+ margin-left: 5px;
+}
+
+#az-za {
+  display: flex;
+  justify-content: center;
+}
+
+main, .main{
+  display: flex;
+  flex: 1;
+  height: 100%;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+  min-height: 100vh;
+} 
+
+.card{
+  display: flex;
+  flex-direction: row;
+  border-bottom: 2px solid rgb(215, 213, 213);
+  max-width: 500px;
+  max-height: 600px;
+  padding: 5px;
+  position: relative;
+}
+
+h2, h3, p{
+  margin:5px 0;
+}
+
+.card img{
+  object-fit: cover;
+  width: 130px;
+  height: 220px;
+  margin: 5px 20px 5px 5px;
+  box-shadow: 3px 3px 5px #AAA;
+}
+
+#rate{
+  position: absolute;
+  margin: 10px;
+  font-size: 0.875em;
+  padding: 10px 10px;
+  color: #fff;
+  border-radius: 25px;
+  background-color: var(--f1);
+}
+
+footer {
+  position: sticky;
+  bottom: 0;
+  height: 40px;
+  color: #fff;
+  background-color: var(--f3);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 10px;
+  font-size: .6em;
+}
+
+.logo {
+  width: 40px;
+  height: auto;
+}
+
+.card div h2{
+  color: var(--f4);
+}
+
+.card div h3{
+  color: var(--f3);
+}
+
+.card div p{
+  color: var(--f2);
+}
+
+/* -----------------------------------------------MEDIA QUERIES--------------------------------------- */
+/* -----------------------------------MOBILE SCREENS 360px to 670xp 1column--------------------------- */
+
+@media (min-width: 360px) and (max-width:670px){
+  button, select {
+    font-size: 0.875em;
+    margin: 5px;
+    padding: 5px 5px;
+    background-color:rgb(255, 255, 255); 
+    color: #2f2f2f;
+    border: none;
+    border-radius: 20px;
+    cursor: pointer;
+  }
+
+  h1{
+    padding: 30px
+  }
+
+}
+/* -----------------------------------TABLET SCREENS 671px to 1024xp 2 colums-------------------------- */
+@media (min-width:671px) and (max-width:1024px) {
+
+
+.sortButtons {
+    width: 80%;
+  }
+}
+
+/* -----------------------------------PC SCREENS >1024px 4 columns ------------------------------------- */
```

Comment:
the comment says that there should be 4 columns, but we can only see 3. Also, maybe the text could be a bit larger?

Reply:
the same here.